osu.Game/Screens/Multi/Match/MatchSubScreen.cs

```diff
@@ -148,12 +152,14 @@ private void load()
 
                 if (tab.NewValue is SettingsMatchPage)
                 {
+                    header.ShowBeatmapPanel.Value = false;
```

Comment:
just my opinion, but I think it would be better to put this at the bottom of line or even do smth like `header.ShowBeatmapPanel = !isSettingsPage;` out of the if block